.kokoro/deps-finder.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Compare shared-dependencies version in a given dependency to desired version.
+# Return 0 for success, 1 for failed to find POM, 2 for shared-deps not present, 3 for incorrect version.
+
+# Options to run:
+# 0 Arguments: Test the latest commit (with a dependency update) for the latest version of shared-deps.
+# 1 Argument: Input artifactId:version to test the given dependency for the latest version of shared-deps.
+# 2 Arguments: Input artifactId:version to test the given dependency for a specific version of shared-dependencies
+# (only input a version number for the second argument).
+
+# Get the directory of the build script
+scriptDir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
+
+# include common functions
+source ${scriptDir}/common.sh
+source ${scriptDir}/shared-deps-helper.sh
+
+# Pass in dependency to script.
+# Dependency should be of the form artifactId:version
+
+function checkLibrary() {
```

Comment:
Please document what the inputs and outputs of this function are.

Reply:
I documented it slightly higher above in the file, but also realized it was right beside the copyright so was very easy to miss. I placed it right on top of the function in the new version!